chrome/renderer/chrome_content_renderer_client.cc

```diff
@@ -1607,7 +1619,34 @@ void ChromeContentRendererClient::WillDestroyServiceWorkerContextOnWorkerThread(
       context, service_worker_version_id, service_worker_scope, script_url);
 #endif
 }
+#if defined(CASTANETS)
+void ChromeContentRendererClient::DidCreateScriptContext(
+    blink::WebFrame* frame,
+    v8::Handle<v8::Context> context,
+    int world_id) {
+  const content::RenderView* render_view =
+      content::RenderView::FromWebView(frame->View());
 
+  if (!widget_)
```

Comment:
How about moving to top of this function?

Reply:
Ok, let me move this to top of this function.